cloud_functions/flagCryptominer.js

```diff
@@ -0,0 +1,35 @@
+const utils = require('./utils')
+const { ValidationError }  = require('./ValidationError')
+
+
+const run = (mixpanel, message) => {
+  try {
+    const userSubjectId = validate(message)
+    if (userSubjectId) {
```

Comment:
It looks like `validate` already ensures that we have a `userSubjectId`, so we probably don't need this condition.

Reply:
Good catch. This is left over from then `validate` just logged instead of throwing an exception.